sui/deploy-gateway.js

```diff
@@ -126,6 +159,14 @@ if (require.main === module) {
     program.addOption(new Option('--minimumRotationDelay <minimumRotationDelay>', 'minium delay for signer rotations (in ms)').default(0));
     program.addOption(new Option('--domainSeparator <domainSeparator>', 'domain separator').default(HashZero));
     program.addOption(new Option('--nonce <nonce>', 'nonce for the signer (defaults to HashZero)'));
+    program.addOption(new Option('--previousSigners <previousSigners>', 'number of previous signers to retain').default(0));
```

Comment:
see the new commander syntax we're using in `sui/gateway.js`. Split deploy/upgrade into two commands and use positional args for mandatory args. We want to switch to a nested CLI command structure

Reply:
Could also tackle this as part of merging gateway deploy functionality into deploy-contract, so it can be deploy/upgrade any contract